Customer Story: Qred (#12132)
PostHog/posthog.com: contents/customers/qred.md

Eventually, Lezgin began the search for a new tool that could better track user behaviour without eating up dev time and while still meeting the strict regulatory requirements the team has to operate under. 

“I follow a lot of open source blogs and that’s where I heard about PostHog,” Lezgin explains. “I really liked that it was an open source SaaS, because that essentially gives me an easy exit plan: if I decide to leave, I can just host it myself.”

Spoiler: Lezgin did not decide to leave PostHog.

<BorderWrapper> <Quote imageSource="/images/customers/lezgin.jpg" size="md" name="Lezgin Zilan" title="CTO, Qred" quote={`“One thing I have to say is that PostHog’s support is awesome. There’s no outsourced first line or call centers — you get real answers fast, straight from the actual engineers. If you find a bug, they show you the GitHub issue and you can literally watch it get fixed. I love that.”`} /> </BorderWrapper>

## More than autocapture: Replacing the whole stack

> It wasn’t just the open-source safety net that made PostHog stand out — he also saw how much more the team could accomplish once the tracking headache was solved with PostHog’s [autocapture](/docs/product-analytics/autocapture).

Reviewer: ```suggestion
It wasn’t just the open-source safety net that made PostHog stand out, he also saw how much more the team could accomplish once the tracking headache was solved with PostHog’s [autocapture](/docs/product-analytics/autocapture).
```

Once Google Analytics was out of the picture for tracking product usage, Lezgin started consolidating other tools into PostHog — swapping out [LaunchDarkly](/blog/posthog-vs-launchdarkly) and other [experimentation](/experiments) and [replay tools](/session-replay). The whole development stack got leaner, cheaper, and more powerful. 

PostHog has since become Qred’s single source of truth for an increasing amount of frontend data — a place where the whole team can work together to make product decisions without bouncing between tools. Even non-technical staff now frequently use PostHog to make decisions. 

<BorderWrapper>
  <div className="bg-[#E5E7E0] dark:bg-[#2C2C2C] rounded-lg p-6 my-8">
    <div className="space-y-4">
      <img 
        src="https://res.cloudinary.com/dmukukwp6/image/upload/qred_screen1_4d8cf22b34.jpg" 
        alt="Qred experiment screenshot" 
        className="w-full rounded-md shadow-sm"
      />
      <h3 className="text-lg font-semibold text-gray-900 dark:text-white mb-4">Building a better onboarding flow with PostHog</h3>
      <p className="text-gray-700 dark:text-gray-300 leading-relaxed">

>         Backed by PostHog as a single source of truth, Qred could make bold changes with confidence. When adding an ID verification step to onboarding, for example, some feared a drop in conversion. However, an A/B test proved it actually boosted retention — unblocking the team and improving the user journey without hurting conversion.

Reviewer 2: Could we change to "Backed by PostHog as a single source of truth, Qred could make bold changes with confidence. Such as the work on finding the optimal flow length and structure via an A/B/C test, for example, some feared a drop in conversion. However, an A/B test proved it actually boosted retention — unblocking the team and improving the user journey without hurting conversion." ?

Author: ```suggestion
        Backed by PostHog as a single source of truth, Qred can make bold changes with confidence. When trying to find the optimal onboarding flow length and structure, for example, some feared a drop in conversion. However, an A/B test proved it actually boosted retention — unblocking the team and improving the user journey without hurting conversion.
```

      </p>
    </div>
  </div>
</BorderWrapper>

## From quick fixes to a single source of truth

> As a bank, Qred is also beholden to additional regulations beyond the usual obligations of [GDPR](/docs/privacy/gdpr-compliance). The Bank Secrecy Act (BSA), for example, imposes certain obligations for tracking users — and Qred therefore doesn’t pass all customer data through to PostHog. Instead, they use reference IDs that correspond to customer details elsewhere and the team join data warehouse tables when needed for deep analysis. 

Reviewer: ```suggestion
As a bank, Qred is also beholden to additional regulations beyond the usual obligations of [GDPR](/docs/privacy/gdpr-compliance). The Bank Secrecy Act (BSA), for example, imposes certain obligations for tracking users, and Qred therefore doesn’t pass all customer data through to PostHog. Instead, they use reference IDs that correspond to customer details elsewhere and the team join data warehouse tables when needed for deep analysis. 
```
